app/controllers/file_controller.rb

```diff
@@ -50,7 +50,9 @@ def show
     disposition = params[:disposition] || 'inline'
     filename = params["filename"] || "#{objid}#{fileid}"
     headers['Content-Disposition'] = "#{disposition}; filename=#{filename}"
-    if ds.mimeType
+    if filename.include?('.jpg')
```

Comment:
@mcritchlow Sure, no problem. I will proceed it with option 2.
Do you get any case that the image got corrupted(can not be displayed) due to the wrong mimetype in its metadata?